Total parameters: 249221
Epoch 1/100
----------
  0%|[34m                                                                                                           [39m| 0/52 [00:00<?, ?it/s]C:\Users\ASUS\anaconda3\envs\gpu_torch_5\Lib\site-packages\torch\nn\functional.py:5476: UserWarning: 1Torch was not compiled with flash attention. (Triggered internally at C:\cb\pytorch_1000000000000\work\aten\src\ATen\native\transformers\cuda\sdp_utils.cpp:263.)
  attn_output = scaled_dot_product_attention(q, k, v, attn_mask, dropout_p, is_causal)












100%|[34m██████████████████████████████████████████████████████████████████████████████████████████████████[39m| 52/52 [00:26<00:00,  1.97it/s]


 97%|[32m███████████████████████████████████████████████████████████████████████████████████████████████▏  [39m| 34/35 [00:05<00:00,  6.69it/s]
Total time:31.649067163467407
Loss: 2.307344733729507
Validation Loss: 2.3049057310277767
HGR Accuracy: tensor(0.0909, device='cuda:0')
ID Accuracy: tensor(0.1000, device='cuda:0')
Validation HGR Accuracy: tensor(0.0909, device='cuda:0')
Validation ID Accuracy: tensor(0.1000, device='cuda:0')
Epoch 2/100
100%|[32m██████████████████████████████████████████████████████████████████████████████████████████████████[39m| 35/35 [00:05<00:00,  6.79it/s]














100%|[34m██████████████████████████████████████████████████████████████████████████████████████████████████[39m| 52/52 [00:30<00:00,  1.72it/s]

100%|[32m██████████████████████████████████████████████████████████████████████████████████████████████████[39m| 35/35 [00:05<00:00,  6.19it/s]
  0%|[34m                                                                                                           [39m| 0/52 [00:00<?, ?it/s]
Total time:35.86602234840393
Loss: 2.305586259726322
Validation Loss: 2.304208149476485
HGR Accuracy: tensor(0.0909, device='cuda:0')
ID Accuracy: tensor(0.1000, device='cuda:0')
Validation HGR Accuracy: tensor(0.0909, device='cuda:0')
Validation ID Accuracy: tensor(0.1000, device='cuda:0')
Epoch 3/100















100%|[34m██████████████████████████████████████████████████████████████████████████████████████████████████[39m| 52/52 [00:30<00:00,  1.68it/s]


 86%|[32m████████████████████████████████████████████████████████████████████████████████████              [39m| 30/35 [00:05<00:00,  5.68it/s]
Total time:36.821279525756836
Loss: 2.3051724766239974
Validation Loss: 2.303437819914384
HGR Accuracy: tensor(0.0909, device='cuda:0')
ID Accuracy: tensor(0.1000, device='cuda:0')
Validation HGR Accuracy: tensor(0.0909, device='cuda:0')
Validation ID Accuracy: tensor(0.1000, device='cuda:0')
Epoch 4/100
100%|[32m██████████████████████████████████████████████████████████████████████████████████████████████████[39m| 35/35 [00:05<00:00,  5.93it/s]















100%|[34m██████████████████████████████████████████████████████████████████████████████████████████████████[39m| 52/52 [00:31<00:00,  1.65it/s]


 94%|[32m████████████████████████████████████████████████████████████████████████████████████████████▍     [39m| 33/35 [00:05<00:00,  5.30it/s]
Total time:37.57931089401245
Loss: 2.304031128160881
Validation Loss: 2.301837488521229
HGR Accuracy: tensor(0.0909, device='cuda:0')
ID Accuracy: tensor(0.1000, device='cuda:0')
Validation HGR Accuracy: tensor(0.0909, device='cuda:0')
Validation ID Accuracy: tensor(0.1000, device='cuda:0')
Epoch 5/100
100%|[32m██████████████████████████████████████████████████████████████████████████████████████████████████[39m| 35/35 [00:06<00:00,  5.73it/s]















100%|[34m██████████████████████████████████████████████████████████████████████████████████████████████████[39m| 52/52 [00:33<00:00,  1.54it/s]



100%|[32m██████████████████████████████████████████████████████████████████████████████████████████████████[39m| 35/35 [00:06<00:00,  5.55it/s]
Traceback (most recent call last):
  File "D:\IIT Delhi\Projects\Adversarial games\Experimentation\codeBase\trainer.py", line 69, in <module>
    train_metrics, val_metrics = train_val(train_dataLoader,
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\IIT Delhi\Projects\Adversarial games\Experimentation\codeBase\epoch.py", line 176, in train_val
    torch.save(model.state_dict(),model_path)
  File "C:\Users\ASUS\anaconda3\envs\gpu_torch_5\Lib\site-packages\torch\serialization.py", line 628, in save
    with _open_zipfile_writer(f) as opened_zipfile:
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ASUS\anaconda3\envs\gpu_torch_5\Lib\site-packages\torch\serialization.py", line 502, in _open_zipfile_writer
    return container(name_or_buffer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ASUS\anaconda3\envs\gpu_torch_5\Lib\site-packages\torch\serialization.py", line 473, in __init__
    super().__init__(torch._C.PyTorchFileWriter(self.name))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: File test.pth cannot be opened.